lab/src/Microsoft.SyndicationFeed/src/Microsoft.SyndicationFeed.csproj

```diff
@@ -3,7 +3,15 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <VersionSuffix>preview</VersionSuffix>
```

Comment:
Yes it will. We could make it _preview1_ if it is more in line with other packages, but _preview_ should be fine as well.